iothub_client/samples/iothub_client_c2d_streaming_sample/iothub_client_c2d_streaming_sample.c

```diff
@@ -88,14 +87,7 @@ static void on_ws_send_frame_complete(void* context, WS_SEND_FRAME_RESULT ws_sen
 {
     (void)context;
 
-    if (ws_send_frame_result == WS_SEND_FRAME_OK)
```

Comment:
Can't this still happen? If no I think it would be nice to have an assert on what result is expected.

Reply:
No matter the result, the sample ends when this function is called. Then there is nothing else to be done.